packages/teams-js/src/public/pages.ts

```diff
@@ -752,4 +752,91 @@ export namespace pages {
       return runtime.supports.pages ? (runtime.supports.pages.appButton ? true : false) : false;
     }
   }
+
+  /**
+   * Provides functions for navigating without needing to specify your application ID.
+   *
+   * @beta
+   */
+  export namespace currentApp {
+    /**
+     * Parameters for the NavigateWithinApp
+     *
+     * @beta
+     */
+    export interface NavigateWithinAppParams {
+      /**
+       * As of 2.0.0, please use {@link app.PageInfo.id | app.Context.page.id} instead
```

Comment:
hmm? I don't think this should be here. Same for the `subPageId` prop below.

Reply:
Do you mean the `@link` code?

Reply:
I mean the whole `As of 2.0.0 ...` bit.

Reply:
Resolved in the latest commit

Reply:
I still don't think these comments are correct. For pageId, I think we want to say that pageId should be set to the value of the page within your app that you want to navigate to. Those ids are defined in your app's manifest file. For subpageId, that value is a developer defined value that they can use to navigate *within* their page. It's not in the manifest, but the value the developer passes to subPageId here will be retrievable from the context object (app.PageInfo.subPageId). Please work with the PM for this feature to make sure the comment is clear and correct.

Reply:
Resolved in the latest commit